cropper/src/test/java/com/canhub/cropper/BitmapUtilsTest.kt

```diff
@@ -0,0 +1,126 @@
+package com.canhub.cropper
+
+import io.mockk.mockkObject
+import io.mockk.unmockkObject
+import org.junit.After
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+
+private val RECT_POINTS: FloatArray = floatArrayOf(1f,2f,3f,4f, 5f, 6f, 7f, 8f)
+private val LOW_RECT_POINTS: FloatArray = floatArrayOf(1f)
+
+class BitmapUtilsTest {
+
+    @Before
+    fun setup() {
+        mockkObject(BitmapUtils)
+    }
+
+    @After
+    fun teardown() {
+        unmockkObject(BitmapUtils)
+    }
+
+    @Test
+    fun `WHEN float array of rectangle points is provided, THEN result should be left value of the bounding rectangle`() {
+        // when
```

Comment:
Let's please use upper case for this comments `// WHEN` I know this is not explained anywhere, sorry for this =/

Reply:
Okay

Reply:
Done